### PR TITLE
Fix docs up, make a ScorePlayer module, & another example

### PR DESCRIPTION
**Pull request by ology**

### CODE REVIEW

File: lib/MIDI/RtMidi/FFI/ScorePlayer.pm [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+
+=head1 SEE ALSO
+
+TBD
+
+=head1 AUTHOR
+
+JBARRETT
+
+=head1 COPYRIGHT AND LICENSE
```
Comment (jbarrett) [inline lib/MIDI/RtMidi/FFI/ScorePlayer.pm:134]: You should make yourself the author / copyright holder - most of `sub play` was pulled from your code as well.

Not sure how PodWeaver / dzil will handle this but we'll find out :)

File: examples/drum-circle.pl [hunk 2]
```diff
@@ -5,9 +5,13 @@
 # Adapted from 'drum-circle' by Gene Boggs
 # https://github.com/ology/Music
 
+use if $ENV{USER} eq 'gene', lib => map { "$ENV{HOME}/sandbox/$_/lib" } qw(MIDI-Drummer-Tiny MIDI-Util Music-Duration-Partition);
```
Comment (ology) [inline examples/drum-circle.pl:8]: I am not sure why these lines are showing up in the PR. They are *not* in the committed and pushed file. WTF :\

**Comment by jbarrett** [comment]
For those playing along at home, this work now lives at [MIDI::RtMidi::ScorePlayer](https://metacpan.org/pod/MIDI::RtMidi::ScorePlayer). Many thanks @ology!

File: examples/drum-circle.pl (hunk 2 above)
Comment (jbarrett) [reply examples/drum-circle.pl]: It's not called "git" because it's nice to use and friendly :)

**Pull request closed without merging**